config/arc_cluster/README.md

```diff
@@ -0,0 +1,37 @@
+# Running open-gira on ARC
+
+## Persistence
+
+Either create or attach to a tmux session:
+`tmux attach-session -t <id>`
+`tmux`
+
+## Resources
+
+Allocate some nodes for use:
+`salloc --ntasks-per-node=<max tasks per node> --nodes=<num nodes> --partition=<short|medium|long> --time=01:00:00 --mem=8000`
+
+## Python environment
+
+We need the snakemake executable to create jobs for us. We could get this from
+the module load system, but their version is quite old (6.10.0). N.B. Versions
+<7.0.0 may cause the following problem:
+https://github.com/snakemake/snakemake/issues/1392
+
+Instead use a conda environment we have created which contains snakemake:
+`conda activate snakemake-7.12.1`
+
+## Osmium
+
+N.B. Some open-gira jobs require the use of osmium. This has been compiled on
+the cluster with `$DATA/osmium/build_osmium.sh`. The executable (actually a
+wrapper) is in ~/bin. N.B. To _run_ osmium, the Boost C++ libraries must be
+available. They are provided by the cluster module system as
+Boost/1.77.0-GCC-11.2.0, and loaded as a step in the wrapper script.
+
+## Invocation
+
+From the open-gira repo call the command we wish to run, using the cluster
+specific profile. For more details on the cluster execution, see the
+config.yaml file in the profile directory.
+`snakemake --profile config/arc_cluster <target name>`
```

Comment:
Add a note on interpreting errors e.g.

```
Error in rule create_overall_bbox:
jobid: 10
output: results/json/tanzania-mini.json
conda-env: /data/ouce-gri-jba/anaconda/envs/1b06bc7edf94bde3731973d2898f9d82
cluster_jobid: 2682538
Error executing rule create_overall_bbox on cluster (jobid: 10, external: 2682538, jobscript: /data/ouce-gri-jba/mert2014/open-gira/.snakemake/tmp.pvgmp5pz/snakejob.create_overall_bbox.10.sh). For error details see the cluster log and the log files of the involved rule(s).
Exiting because a job execution failed. Look above for error message
Complete log: .snakemake/log/2022-10-05T181802.083378.snakemake.log
```

This reports an error in the rule `create_overall_bbox` so we should look in `logs/create_overall_bbox`

```
ls logs
ls logs/create_overall_bbox
less logs/create_overall_bbox/create_overall_bbox-DATASET\=tanzania-mini\,OUTPUT_DIR\=results-2682538.out `
```

Which prints the full log for that job, where we can look for a traceback:

```
Traceback (most recent call last):
  File "/data/ouce-gri-jba/mert2014/open-gira/.snakemake/scripts/tmp5ruqzjkn.create_overall_bbox.py", line 24, in <module>
    bboxes = subprocess.check_output(["osmium", "fileinfo", osm_file, "-g", "header.boxes"])
  File "/data/ouce-gri-jba/anaconda/envs/1b06bc7edf94bde3731973d2898f9d82/lib/python3.9/subprocess.py", line 424, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/data/ouce-gri-jba/anaconda/envs/1b06bc7edf94bde3731973d2898f9d82/lib/python3.9/subprocess.py", line 505, in run
    with Popen(*popenargs, **kwargs) as process:
  File "/data/ouce-gri-jba/anaconda/envs/1b06bc7edf94bde3731973d2898f9d82/lib/python3.9/subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/data/ouce-gri-jba/anaconda/envs/1b06bc7edf94bde3731973d2898f9d82/lib/python3.9/subprocess.py", line 1821, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'osmium'
```

In particular, here, the `FileNotFoundError` says that the job runner couldn't find `osmium`, which is needed to run this rule.